docker/Dockerfile.copy

```diff
@@ -1,5 +1,7 @@
 FROM debian:latest
 
+ARG environment
```

Comment:
Maybe we want to set `k8s-local` as default here also?
```suggestion
ARG environment=k8s-local
```

Reply:
The default value is applied throughout the script — the subsequent `ARG` declarations just bring the existing argument into scope.  See: https://stackoverflow.com/a/53682110 :)

Reply:
Good idea — done! :)